assets/simple-boxes.js

```diff
@@ -307,7 +308,7 @@ window.simpleBoxes._.methods = {
     let overWhichBox = await window.simpleBoxes._.methods.identifyBox(handle,myX,myY);
 
     if(state.mouseDown){
-      // Dragging
+      console.log("Dragging");
```

Comment:
i propose we remove these log lines (here and below) before we merge to main.

Reply:
(or comment them out would be fine too .. if we feel like we may need them for further debugging)